icu4c/source/test/testdata/localeMatcherTest.txt

```diff
@@ -0,0 +1,1959 @@
+# © 2017 and later: Unicode, Inc. and others.
```

Comment:
I presume this is identical to the java version. Side issue: should we have a separate project to contain all the supposed-to-be-identical test files?

Reply:
Yes, we should, and we have discussed it a bit, but that would require significant changes in test code, and may touch on a unified C++/Java build. Not something I want to tackle here. Not the first duplicated test data file, either. At least with identical files it is easy to run the same set of test cases in both C++ and Java.